crawl-ref/source/evoke.cc

```diff
@@ -574,36 +512,6 @@ void wind_blast(actor* agent, int pow, coord_def target)
         else
             mpr("The monsters around you are blown away!");
     }
```

Comment:
The messaging is broken now. We get:
```
You slam into a wall!
You are blown backwards!
```

Which is, appropriately, backwards. :)

This is actually quite tricky to fix properly. We have a few options:
- Defer collisions and handle them separately. This is a bit scary once dispersal traps get involved, but it does mean that we can also remove the message-printing logic entirely.
- Rewrite `::collide()` to be optionally silent and handle printing later. This is a bit hairy as wel.
- Just let wind_blast print messages for each individually affected creature, removing its special printing logic and letting `::knockback()` handle everything. This is fine for monsters - they rarely blow back much more than the player. It would be a bit worse for the Storm card, since that's more likely to blow back many things (and hence be quite message-spammy), but that's going to be spammy anyway once collisions get involved. So let's just do that.